playbooks/roles/consul/defaults/main.yml

```diff
@@ -17,6 +19,23 @@ consul_enable_syslog: true
 
 consul_encrypt: ""
 
+consul_acl_enabled: false
```

Comment:
Yeah I think so, though I had left it off in case enabling this somehow had bad effects on the Open edX VMs handled by Ocim. I don't think it will so we can enable it by default.